docs/TypeScript.md

```diff
@@ -333,12 +335,20 @@ This plugin guide is for Fastify plugins written in JavaScript. The steps outlin
   }
   
   module.exports = fp(myPlugin, {
-    fastify: '3.x'
+    fastify: '3.x',
+    name: 'my-plugin' // this is used by fastify-plugin to derive the property name
   })
   ```
 5. Open `index.d.ts` and add the following code:
   ```typescript
-  // Not necessary, but exporting whatever extra types is supported 
+  import { FastifyPlugin } from 'fastify'
+
+  interface PluginOptions {
+    //...
+  }
+
+  // Optionally, you can add any additional exports
+  // here we are exporting the decorator we added
```

Comment:
If this is intended to be a separate sentence, might make sense to capitalize first letter for consistency, and maybe add dot into previous one